regression/scripts/airfoil_import/airfoil_import_test.py

```diff
@@ -68,8 +67,20 @@ def main():
     for j in range(0, len(airfoil_geometry_3.camber_coordinates)):
         assert( np.abs(airfoil_geometry_3.camber_coordinates[j] - airfoil_geometry_4.camber_coordinates[j]) < 1E-8 )
 
-    plot_airfoil(airfoil_geometry_with_selig[0])
-    
+    # Multiple meshes use too much memory on AppVeyor
+
+    A_MASK_1 = convert_airfoil_to_meshgrid(airfoil_geometry_1)
```

Comment:
do these need to be commented out? If so leave a note about why

Reply:
The comment above the block about having multiple meshes is why they're commented out - they all run and pass on my machine, but AppVeyor runs out of memory. I'm leaving them in place so I can check/verify they still work before pushing any changes.